.github/workflows/ci.yml

```
@@ -6,7 +6,8 @@ on:
 
 jobs:
   test-distcheck:
```

actionlint (1.7.12)

[error] "runs-on" section is missing in job "test-distcheck"

(syntax-check)